snuba/query/allocation_policies/__init__.py

```diff
@@ -856,6 +689,9 @@ def _update_quota_balance(
     def storage_key(self) -> StorageKey:
         return self._storage_key
 
+    def component_namespace(self) -> str:
+        return "allocation_policy"
+
```

Comment:
This is going to ignore all existing allocation policy configurations. Make sure they are preserved